website/docs/cloud-docs/api-docs/organizations.mdx

```diff
@@ -598,22 +598,23 @@ curl \
+| `data.attributes.owners-team-saml-role-id`                                | string  | none     | Optional. SAML only. The name of the ["owners" team](/terraform/enterprise/saml/team-membership#managing-membership-of-the-owners-team)                                                                                                                                                                                                                                            |
+| `data.attributes.assessments-enforced`                                    | boolean | `false`  | Whether or not to compel health assessments for all eligible workspaces. When set to `true`, Terraform performs health assessments on all compatible workspaces, regardless of the value of the workspace setting `assessments-enabled`. When set to `false`, Terraform only performs health assessments for workspaces that opt in by setting `assessments-enabled: true`.        |
+| `data.attributes.allow-force-delete-workspaces`                           | boolean | `false`  | Whether workspace administrators can delete workspaces with resources under management. Refer to the [general organization settings documentation](/terraform/cloud-docs/users-teams-organizations/organizations#general) for additional information. When set to `false, only organization owners may delete these workspaces.                                                    |
+| `data.attributes.default-execution-mode`                                  | string  | `remote` | Which [execution mode](/terraform/cloud-docs/workspaces/settings#execution-mode) to use by default. Valid values are `remote`, `local`, and `agent`.                                                                                                                                                                                                                               |
+| `data.relationships.default-agent-pool.data.id`                           | string  |          | Required when `default-execution-mode` is set to `agent`. The ID of the agent pool belonging to the organization. Do _not_ specify this value if you set `execution-mode` to `remote` or `local`.                                                                                                                                                                                  |
+| `data.relationships.default-project.data.id`                              | string  |          | Specifies the ID of the default project. If set to `null`, Terraform removes the organization's default project. If omitted, Terraform doesn't change the organization's default project. You must have [owner permissions](/terraform/cloud-docs/users-teams-organizations/permissions#organization-owners) to update the default project.                                        |
```

Comment:
@trujillo-adam 👋🏽 I updated the `data.relationships.default-project.data.id` description, what do you think?

Reply:
> @trujillo-adam 👋🏽 I updated the `data.relationships.default-project.data.id` description, what do you think?

Would it still accurate to say that the parameter defaults to to the existing ID if omitted? Maybe "If omitted, Terraform defaults to the existing organization ID."

To me, it is a little awkward to describe what does not happen when you don't set the parameter--sort of like saying that if you don't move, you'll stay in one place. The other bit of feedback is that if we keep this wording fully spell out "does not" instead of using the contraction, which is a style guide violation.

Reply:
The ID is the default project ID so it would have to say something like:

"If omitted, Terraform defaults to the existing default project ID."

The most important thing we need to note here is that if they set the relationship to `null`, it removes the default project. I'm flexible on whether we need to state what happens if it's omitted.